### PR TITLE
handel bower links

### PR DESCRIPTION
**Pull request by wheresrhys**
@i-like-robots

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -192,13 +200,11 @@ class Configurator {
 		// TODO: these modules need to be transpiled
 		// NOTE: these are npm modules, babel has a freak out if you just include all of /node_modules/
 		const es6Modules = [
-			'@financial-times/n-card',
-			'@financial-times/n-image',
-			'@financial-times/n-section',
-			'@financial-times/n-ui'
+			'@financial-times/n-'
```
Comment (wheresrhys) [inline lib/index.js]: @ironsidevsquincy Should be safe ??

Comment (ironsidevsquincy) [inline lib/index.js]: yeah? Does mean it'll try and 'babelify' all n- modules, might impact compile time

Comment (wheresrhys) [inline lib/index.js]: but only n- npm modules that are required by the client side code... which is never gonna be that big a list and I can't see that we'd want any such modules to be excluded from babel

Comment (ironsidevsquincy) [inline lib/index.js]: I mean n- modules that are already babelified will still be parsed, etc unnecessarily by babelify.

Comment (wheresrhys) [inline lib/index.js]: Are there any?

Comment (ironsidevsquincy) [inline lib/index.js]: They should all be, tbh

**Pull request merged**